.gitignore

```diff
@@ -101,9 +101,6 @@ unlinked_spec.ds
 
 # macOS
 **/macos/Flutter/GeneratedPluginRegistrant.swift
```

Comment:
This is wrong too. Only Flutter/ephemeral (which is in macos/.gitignore) is designed to be ignored on macOS. That way we can always add new files to it and control whether they are or are not checked in even for existing projects, to avoid issues I saw with mobile.

Reply:
I think that file needs to move to ephemeral then, it is generated:
```
//
//  Generated file. Do not edit.
//

import FlutterMacOS
import Foundation


func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
}
```

Reply:
Filed https://github.com/flutter/flutter/issues/71762, let's deal with that separately.

Reply:
For the reasons I gave in the bug, I am not confident that this PR will meet the started goal (having those be buildable on macOS) in all cases without fixing this.

Reply:
This project, at least, is buildable on macOS with just this change.